collections/_posts/2024-12-02-SSL_comparison.md

```diff
@@ -0,0 +1,123 @@
+* DINO: named after self-distillation with no labels, several transformations of an image passed through student and teacher networks, similarity of the features extracted is computed as loss, and again the teacher weights are updated with an exponential moving average of the student parameters.
+* Barlow Twins: introduction of an objective function to make as close as possible to identity matrix the cross-correlation matrix between the outputs of two identical networks fed with distorted versions of the same image to reduce redundancy in the components of the vectors.
+
+## Datasets and training parameters
+
+Four datasets with different characteristics are used to compare the 16 methods.
+
+| Dataset     | Image type                  |  Number of classes | Resolution |
+|-------------|-----------------------------|--------------------|------------|
+| TissueMNIST | Kidney cortex cells         | 8                  | 28*28      |
+| PathMNIST   | Colorectal cancer histology | 9                  | 28*28      |
+| TMED-2      | Echocardiogram US images    | 4                  | 112*112    |
+| AIROGS      | Retina color fundus         | 2                  | 384*384    |
+
+The details of the images' distribution are in the following table, with the number of images labeled or not by category. The two first datasets are entirely labeled, so a part of the images were considered as unlabeled to fit the training paradigm of SSL.
+
+![](/collections/images/SSL_comparison/datasets_description.jpg)
+
+Here are some images examples.
+
+![](/collections/images/SSL_comparison/datasets_example.jpg)
+
+# Experimental setup
+
+For the low resolution images ResNet-18 is used, WideResNet-28-2 is used for TMED-2 and ResNet-18 or 50 are tested for AIROGS dataset. \
+A maximum of 200 epochs is performed and training is stopped if the balanced accuracy plateaus for 20 consecutive epochs. 
+
+Hyperparameters (learning rate, weight decay and unlabeled loss weight, in addition to the parameters specific for each method) optimization is made for each method and dataset. To fit with real life constraints, they fixed a certain number of hours with a NVIDIA A100 GPU for each hyperparameter optimization (25h for PathMNIST, 50h for TissueMNIST and 100h for TMED-2 and AIROGS). Within this fixed budget, a random search of hyperparameters is done, tacking the best with validation set.
+
+# Results
+
+Five separate trials were performed and mean balanced accuracy is calculated each 30 or 60 minutes on validation and test sets.
+
+Balanced accuracy on test set over time is represented for each dataset and method. As there is a rough monotonic improvement in test performance over time despite using a realistic-size validation set, they conclude that checkpoint selection and hyperparameters optimization can be effective with a realistically-sized validation set.
```

Comment:
"As there is a rough monotonic improvement in test performance..." → Suggested: " As a roughly monotonic improvement in test performance is observed..."